packages/ui/src/ui/store/thunkDispatch.ts

```diff
@@ -3,6 +3,6 @@ import {ThunkDispatch} from 'redux-thunk';
 import {RootState} from './reducers';
 import {useDispatch} from 'react-redux';
 
-export type AppThunkDispatch = ThunkDispatch<RootState, any, AnyAction>;
+export type AppThunkDispatch<T extends AnyAction = AnyAction> = ThunkDispatch<RootState, any, T>;
```

Comment:
I think it is better to get rid of default value `= AnyAction` here, and set the parameter always explicitly.